server/src/api/bill_exec.py

```diff
@@ -53,12 +53,35 @@ def create_bill(self):
         
         try:
             username = cherrypy.session["username"]
+            #Bill has a title
+            if ((len(title) < 1)):
+                raise ValueError("no title","title")
```

Comment:
`ValueError` could be triggered by some real bug in the code. Should we instead create our own error class, perhaps a subclass of `ValueError`, and only catch this subclass?